js/ui/file_manager/ui.file_manager.js

```diff
@@ -127,6 +127,7 @@ class FileManager extends Widget {
             onError: ({ error }) => this._showError(error),
             onSelectionChanged: this._onItemViewSelectionChanged.bind(this),
             onSelectedItemOpened: ({ item }) => this._tryOpen(item),
+            onSelectedFileOpened: this._createActionByOption("onSelectedFileOpened"),
```

Comment:
We can use existing `onSelectedItemOpened `option for this task

Reply:
repaired